e2e/homeScreen.spec.js

```diff
@@ -47,7 +47,7 @@ describe('Home Screen', () => {
     await Helpers.swipe('wallet-screen', 'up', 'slow', 0.4);
     await Helpers.checkIfVisible('profile-sticky-header');
     await Helpers.checkIfExistsByText(RAINBOW_TEST_WALLET);
-    await Helpers.swipe('wallet-screen', 'down');
+    await Helpers.swipe('wallet-screen', 'down', 'slow', 0.4);
```

Comment:
we were observing the "swipe down" not successfully being able to swipe down unless we added the slow param